.github/workflows/_func.yaml

```diff
@@ -49,6 +49,16 @@ on:
         required: false
         type: boolean
         default: false
+      ignore-changes:
```

Comment:
I think that skipping logic should be integrated only to `pr.yaml`.

Reply:
However I realized that if do not want to skip whole workflow run we will need to do it in _func.yaml and _lint-unit.yaml or rewrite them as composite actions. We could discuss it later today.

Reply:
Sure, let's discuss this later. I was also considering rewriting func and lint-unit tests to composite actions because with reusable workflows, we could run into nesting limitations in the future.